phpstan.neon

```diff
@@ -741,5 +741,31 @@ parameters:
                 - packages/changes-reporting/src/Output/JsonOutputFormatter.php # 14
                 - src/Console/Output/RectorConfigurationFormatter.php # 13
 
+        -
+            message: '#Do not use scalar or array as constructor parameter\. Use "Symplify\\PackageBuilder\\Parameter\\ParameterProvider" service instead#'
```

Comment:
Here I only changed the message, so it should be updatd in the other place in `phpstan.neon`.

```diff
-Do not use scalar or array as constructor parameter\. Use "ParameterProvider" service instead
+Do not use scalar or array as constructor parameter\. Use "Symplify\\PackageBuilder\\Parameter\\ParameterProvider" service instead
```

Reply:
implemented.